cmd/commands/service/command.go

```diff
@@ -66,13 +67,9 @@ func NewCommand(licenseCommandName string) *cli.Command {
 
 	stopCommand := func() error {
 		errorServiceManager := di.ServiceManager.Kill()
-		errorNode := di.Node.Kill()
 		di.Shutdown()
 
-		if errorServiceManager != nil {
-			return errorServiceManager
-		}
-		return errorNode
+		return errorServiceManager
```

Comment:
We need to stop all the dependencies. Currently `Node` is not stopped

Reply:
Added it to the `di.Shutdown()`